client/src/components/ErrorMessageComponent/ErrorMessage.js

```diff
@@ -2,13 +2,15 @@
 // Licensed under the MIT license.
 
 import React, { Component } from "react";
-import { DefaultButton, Spinner } from "office-ui-fabric-react";
+import { DefaultButton, Spinner, MessageBar } from "office-ui-fabric-react";
 import ModalComponent from "../ModalComponent/ModalComponent";
 import { eventService } from "../../services/EventService";
 import { CUSTOM_AUTH_ERROR_MESSAGE, MORE_INFORMATION_LINK, CUSTOM_NOT_FOUND_ERROR_MESSAGE, CUSTOM_AZURE_ERROR_MESSAGE, AUTH_SUCCESS_MESSAGE,
   AUTH_CONFLICT_MESSAGE, AUTH_FORBIDDEN_MESSAGE, AUTH_NOT_FOUND_MESSAGE } from "../../services/Constants";
 import { print } from "../../services/LoggingService";
 import { rbacService} from "../../services/RBACService";
+import { withTranslation } from "react-i18next";
```

Comment:
We _tend_ to put the npm imports above the file imports.

Reply:
Ready